### PR TITLE
Task-58285: Remove Google and Outlook agenda connectors from the agenda codebase

### PR DESCRIPTION
**Pull request by abenmbarek**
Prior to these changes, all connectors are registered and managed using the same module.
After these changes, we will be able to manage all connectors (enable/disable) in a separative way and reduce the strong coupling between them.

### CODE REVIEW

**Review by azayati — CHANGES_REQUESTED** [review]
Can you clean the other files ?

File: agenda-webapps/src/main/resources/locale/portlet/Agenda_en.properties [hunk 1]
```diff
@@ -150,8 +150,6 @@ agenda.disconnect=Disconnect
 agenda.connectYourPersonalAgenda=Connect your personal Calendar
```
Comment (azayati) [inline agenda-webapps/src/main/resources/locale/portlet/Agenda_en.properties:150]: Can you check also if these labels need to be removed:
- agenda.details.calendar.google=Google Calendar
- agenda.details.calendar.office=Microsoft Outlook

Otherwises, the labels in all languages should be removed to make crowdin removing them also

File: agenda-webapps/webpack.prod.js [hunk 2]
```diff
@@ -28,7 +28,6 @@ const config = {
     agendaSettings: './src/main/webapp/vue-app/agenda-user-setting/main.js',
     agendaTimeline: './src/main/webapp/vue-app/agenda-timeline/main.js',
     agendaSearchCard: './src/main/webapp/vue-app/agenda-search/main.js',
-    agendaConnectors: './src/main/webapp/vue-app/agenda-connectors/main.js',
```
Comment (azayati) [inline agenda-webapps/webpack.prod.js:31]: keep that until moving exchange connector, rename it exchangeConnector

File: agenda-webapps/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 3]
```diff
@@ -212,16 +212,7 @@
     </depends>
   </module>
 
-  <module>
-    <name>agendaConnectors</name>
```
Comment (azayati) [inline agenda-webapps/src/main/webapp/WEB-INF/gatein-resources.xml:216]: keep that until moving exchange connector, rename it exchangeConnector

File: agenda-webapps/src/main/webapp/WEB-INF/conf/configuration.xml [hunk 4]
```diff
@@ -10,7 +10,6 @@
   <import>war:/conf/agenda/spaces-templates-configuration.xml</import>
   <import>war:/conf/agenda/job-configuration.xml</import>
   <import>war:/conf/agenda/reminders-listener-configuration.xml</import>
-  <import>war:/conf/agenda/connectors-configuration.xml</import>
```
Comment (azayati) [inline agenda-webapps/src/main/webapp/WEB-INF/conf/configuration.xml]: keep that until moving exchange connector

**Review by azayati — CHANGES_REQUESTED** [review]
see review

File: agenda-webapps/src/main/resources/locale/portlet/Agenda_en.properties (hunk 1 above)
Comment (abenmbarek) [reply agenda-webapps/src/main/resources/locale/portlet/Agenda_en.properties:150]: I confirm it, as you mentioned these labels are no longer used.

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**